examples/spire/k8s-workload-registrar-cluster-role.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: k8s-workload-registrar-role
+rules:
+  - apiGroups: [""]
+    resources: ["endpoints", "nodes", "pods"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["spiffeid.spiffe.io"]
+    resources: ["spiffeids"]
+    verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]
+  - apiGroups: ["spiffeid.spiffe.io"]
+    resources: ["spiffeids/status"]
+    verbs: ["get", "patch", "update"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: k8s-workload-registrar-role-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: k8s-workload-registrar-role
+subjects:
+  - kind: ServiceAccount
+    name: spire-server
+    namespace: spire
```

Comment:
Do we need to keep this in our repository?

Have spire kustomization file in their examples repo?

Reply:
They don't have separate k8s-workload-registrar files in their repo, but they hace a single file with complete server config. https://github.com/spiffe/spire/blob/main/support/k8s/k8s-workload-registrar/mode-crd/config/spire-server-registrar.yaml